contrib/release.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const child_process = require('child_process');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function prompt(question) {
+  return new Promise(resolve => {
+    readline.question(`${question}: `, response => {
+      readline.close();
+      resolve(response.trim());
+    });
+  });
+}
+
+async function run() {
+  const [, , VERSION] = process.argv;
+
+  if (!VERSION) {
+    throw new Error('error: must specify a valid version');
+  }
+
+  const githash = child_process
+    .execSync('git rev-parse HEAD')
+    .toString()
+    .trim();
+
+  const dir = child_process
+    .execSync(
+      `aws s3 ls realm-clis --recursive | grep ${githash} | sort | tail -n 1 | awk '{print $4}' | cut -f 1-1 -d "/"`
+    )
+    .toString()
+    .trim();
+
+  if (!dir) {
+    throw new Error(`error: failed to find release for git hash ${githash}`);
+  }
+
+  const currentData = JSON.parse(
+    child_process
+      .execSync(`aws s3 cp 's3://realm-clis/versions/cloud-prod/CURRENT' -`)
+      .toString()
+      .trim()
+  );
+
+  const updatedData = JSON.stringify(
+    {
+      version: VERSION,
+      info: {
+        'linux-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/linux-amd64/realm-cli`,
+        },
+        'macos-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/macos-amd64/realm-cli`,
+        },
+        'windows-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/windows-amd64/realm-cli.exe`,
+        },
+      },
+      past_releases: [
+        {
+          version: currentData.version,
+          info: currentData.info,
+        },
+        ...(currentData.pastReleases || []),
+      ],
+    },
+    null,
+    '  '
+  );
+
+  console.info(
+    "uploading the following JSON file to S3 bucket 'realm-clis/versions/cloud-prod/CURRENT':\n",
+    updatedData
+  );
+
+  const response = await prompt('proceed? [y/n]');
+  if (response !== 'y') {
+    console.info('update canceled');
+    return;
+  }
+
+  const uploadResult = child_process
```

Comment:
[nit] `uploadResult` => `uploadErr`?

Reply:
this isn't necessarily an err, this is just the actual bash output. if there is an actual error (i.e. a non zero exit code), the `execSync` call will throw, but there are also cases where the call is "successful" but something happens preventing the file data from being uploaded. in that case, we'll get some text output here.

Reply:
roger that, makes sense to me!